ha/monitor/k8s/parser.py

```diff
@@ -141,11 +146,13 @@ def _create_health_alert(self, res_type, res_name, health_status, generation_id)
         health_status: health of that resource. Ex: online, failed
         generation_id: name of the node in case of node alert
         """
-        self.event = HealthEvent()
-        self.event.set(HealthAttr.RESOURCE_TYPE.value, res_type)
-        self.event.set(HealthAttr.RESOURCE_ID.value, res_name)
-        self.event.set(HealthAttr.NODE_ID.value, res_name)
-        self.event.set(HealthAttr.RESOURCE_STATUS.value, health_status)
+        self.payload[HealthAttr.RESOURCE_TYPE.value] = res_type
+        self.payload[HealthAttr.RESOURCE_ID.value] = res_name
+        self.payload[HealthAttr.NODE_ID.value] = res_name
+        self.payload[HealthAttr.RESOURCE_STATUS.value] = health_status
+        # Create event schema object with payload data also
+        # can change value of payload attributes with set function
+        self.event = HealthEvent(**self.payload)
```

Comment:
Here cIuster_id will be set to "". Is  the logic for populating the correct cluster_id already present in fault_tolerance ?

Reply:
In fault_monitor,  process_message function set EventAnalyzer, where in EventAnalyzer set attribute of cluster_id with 
 health_event = self._cluster_resource_parser.parse_event(msg)
that set cluster_id @Madhura-08  please confirm .
https://github.com/Seagate/cortx-ha/blob/7d7489b415e3a566b5c0ab8f82a6e1c39c4b0470/ha/core/event_analyzer/parser/parser.py#L167

Reply:
Yes..This is correct.